### PR TITLE
Add section on setting up git in RStudio for Mac OS X and Windows users

### PR DESCRIPTION
**Pull request by pschloss**
This addresses Issue #93

### CODE REVIEW

**Comment by pschloss** [comment]
Hey @naupaka, let me know what you think of this PR. I think it addresses the issues we ran into back in March.

**Comment by tomwright01** [comment]
This looks great to me, I'd like someone with a mac to take a quick look before merging.

File: instructors.md [hunk 1]
```diff
@@ -6,11 +6,31 @@ subtitle: Instructor's Guide
...
-installing things (even if you asked students to install in advance, longer if 
+Leave about 30 minutes at the start of each workshop and another 15 mins
+at the start of each sesson for technical difficulties like WiFi and
+installing things (even if you asked students to install in advance, longer if
 not).
 
+
+## Setting up git in RStudio
+
+There can be difficulties linking git to RStudio depending on the operating system and the version of the operating system. To make sure git is properly installed, the learners should go to the Options window.
```
Comment (naupaka) [inline instructors.md]: I might change from 
"...properly installed, the learners should go to the Options window." 
to 
"... properly installed and configured, the learners should go to the Options window in the RStudio application:"

File: instructors.md [hunk 2]
```diff
@@ -6,11 +6,31 @@ subtitle: Instructor's Guide
...
+
+* **Mac OS X:**
+  * Go RStudio -> Preferences... -> Git/SVN
+  * Check and see whether there is a path to a file in the "Git executable" window. If not, the next challenge is figuring out where Git is located.
+  * In the terminal enter `which git` and you will get a path to the git executable. In the "Git executable" window you may have difficulties finding the directory since OS X hides many of the systems file. If the user presses "Command-Shift-G" that will allow you to enter `/usr/bin/git` or whatever the path is.
```
Comment (naupaka) [inline instructors.md]: "systems file" -> "operating system files"

Comment (naupaka) [inline instructors.md]: Change from 
"If the user presses "Command-Shift-G" that will allow you to enter `/usr/bin/git` or whatever the path is."
to
"While the file selection window is open, pressing "Command-Shift-G" will pop up a text box where you will be able to enter `/usr/bin/git` or whatever the path to your git executable might be."

File: instructors.md [hunk 3]
```diff
@@ -6,11 +6,31 @@ subtitle: Instructor's Guide
...
+* **Windows:**
+	* Go Tools -> Global options... -> Git/SVN
+	* If you use the Software Carpentry Installer, then 'git.exe' should be installed at 'C:/Program Files/Git/bin/git.exe'
+
+To prevent the learners from having to re-enter their password each time they push a commit to GitHub, this command will make it so they only have to enter their password once:
```
Comment (naupaka) [inline instructors.md]: "this command will" -> "this command (which can be run from a bash prompt) will"

**Comment by naupaka** [comment]
Thanks @pschloss for the PR on this! I made a couple wording suggestions, but otherwise looks good to merge.

**Comment by pschloss** [comment]
Thanks for the feedback, @naupaka and @tomwright01 - I've made he suggested edits. Hopefully, people will find this contribution helpful!

**Comment by naupaka** [comment]
Looks good. Thanks once again @pschloss. I think it's great to have these tips available with the lesson materials.

**Pull request merged**